### DISCUSSION TITLE
Possible exploit using riverside data management testing I need some more help explaining

### DISCUSSION THREAD

**Discussion by U-Nazu** [3kh0/ext-remover#1008] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Saw a kid using riverside data management testing PLEASE HELP EXPLAINING IT MORE
He did as so:
1. Use Riverside data test
2. Typed youtube.com in the first log in box
3. Then left clicked on it then just maybe enter i think.
Is this a new exploit or what please help and tysm if you do!
Note: he got able to go to google or something

**Comment by xoidex** [comment]
pretty sure this is a webview exploit but don't take my word for it
